tools/nni_cmd/nnictl_utils.py

```diff
@@ -681,45 +681,64 @@ def monitor_experiment(args):
     set_monitor(False, args.time)
 
 def export_trials_data(args):
-    '''export experiment metadata to csv
+    '''export experiment metadata and intermediate results to json or csv
     '''
+    def groupby_trial_id(intermediate_results):
+        sorted(intermediate_results, key=lambda x: x['timestamp'])
+        groupby = dict()
+        for content in intermediate_results:
+            groupby.setdefault(content['trialJobId'], []).append(eval(content['data']))
```

Comment:
why use eval()?

Reply:
The returned data is serialized. For example, `"\"{\\\"default\\\": 93.64, \\\"other_metric\\\": 2.0}\""`.
Using `eval()` can easily handle the `\` part and reconstruct the data structure from the string. It is troublesome for users to handle serialized data.
The data from the server can be trusted so I think using `eval` here is safe here.

Reply:
why not simply use json.loads?

Reply:
It looks that `json.loads()` is a good solution.

Reply:
please double check whether it works properly

Reply:
Sure I will. Thanks for reminding me.